dts/bindings/can/can-controller.yaml

```diff
@@ -46,6 +46,11 @@ child-binding:
       };
 
   properties:
+    min-bitrate:
+      type: int
+      description: |
+        The minimum bitrate supported by the CAN transceiver in bits/s.
```

Comment:
```suggestion
        The minimum bitrate supported by the CAN controller in bits/s.
```

Reply:
No, this property is for the CAN transceiver child node.